lean4/galois/init/io.cpp

```diff
@@ -0,0 +1,153 @@
+
+#include <fcntl.h>
+#include <stdio.h>
+#include <unistd.h>
+
+#include "apply.h"
+
+using namespace lean;
+
+////////////////////////////////////////////////////////////////////////
+// Copied from io.cpp
+
+extern obj_res const REAL_WORLD = box(0);
+
+static obj_res set_io_error(obj_arg r, obj_arg e) {
```

Comment:
I noticed `set_io_error` and `set_io_result` are now exported from `io.h`, so we do not need to define them outselves.